nebula/ui/components/VPNControllerView.qml

```diff
@@ -580,7 +580,7 @@ Item {
                 return;
             }
             Sample.settingsViewOpened.record();
-            VPN.settingsNeeded();
+            mainStackView.push("qrc:/ui/views/ViewSettings.qml", StackView.Immediate)
```

Comment:
why this change?

Reply:
Because if a tutorial is playing and we call VPN.settingsNeeded(), the 'leave tutorial?' modal will be triggered. Maybe it would be better to add an optional param to `onSettingsNeeded()` to distinguish between calls from clicks to the Settings button and opens from the top bar menu?

Reply:
Yes! This is a good idea. Let's add the parameter.

Reply:
Going to merge this while tests are green and will do this in a fast (later today) follow to avoid touching a bunch of new files.